packages/docusaurus-1.x/lib/server/utils.js

```diff
@@ -53,6 +53,7 @@ function minifyCss(cssContent) {
     .process(cssContent, {
       preset: 'default',
       zindex: false,
+      from: undefined,
```

Comment:
Note: `from: undefined` is also set on `autoprefixer`:

https://github.com/facebook/docusaurus/blob/a718dcd73dc0a12f67d7767c498d62c3f57c2e99/packages/docusaurus-1.x/lib/server/utils.js#L61-L65